docs/stencil-docs/reference-docs/front-matter-reference.md

```diff
@@ -255,12 +261,44 @@ gql: "query productById($productId: Int!) {
   }
 }"
 ```
-  
+
+<!--
+type: tab
+title: Category
+-->
+
+ ```yaml title="Front matter block in category.html template" lineNumbers
+category:
+    shop_by_price: true
+    products:
+        limit: {{theme_settings.categorypage_products_per_page}}
+gql: "query categoryById ($categoryId: Int!){

```

Comment:
yeah, testing is a good idea. are we sure that the `gql` attribute is all the way left?